### ISSUE TITLE
i18n: two locales inheriting from a base locale

### ISSUE THREAD

**Issue by vshulev** [middleman/middleman#2005]
Apologies if this doesn't belong here, feel free to close it. I tried [asking on Stackoverflow](http://stackoverflow.com/questions/40555228/middleman-is-it-possible-for-one-locale-to-inerit-data-from-another-locale) but I didn't get an answer.

**Summary of the issue:** Our website is available in English, but we would like to support a US and a UK version both with small differences between them, but both essentially having 90% overlapping translations. Is it possible to define a base `en` language and two variants `en_gb` and `en_us` which both inherit from the base translation?

**Comment by joallard** [comment]
Can you show a very small example of what you've tried and how you'd ideally like it to work?

**Comment by vshulev** [comment]
I don't really know any Ruby so I haven't tried writing any custom code yet. What I did in my config file was instead of defining the language as :en I defined `:en_gb` and `:en_us`. However that way instead of having a single `en.yml` I have `en_gb.yml` and `en_us.yml` which contain almost identical data.

Ideally, there would be a way to define in the config file that en is the root locale and en_us and en_gb are child locales. Then we would have `en.yml` containing most of the translations and `en_gb.yml`, `en_us.yml` containing only the translations that should be overwritten.

I'm a bit confused about how to approach this problem. Can it be done in the config file, or do I need to write a custom extension for this?
